UKHomeOffice/cerberus-service
# COP-9675: Add tests to verify different RoRO Tourist task details

## Description
Add automation tests to verify the task summary details of different RoRo Tourist task details.

## To Test
npm run cypress:runner

run following tests from spec `create-tasks.spec.js`
`Should create a RoRo task with payload contains multiple passengers`
`Should create a RoRo task with payload contains Single passenger`
`Should verify the RoRo Tourist task with Vehicle Details`

## Developer Checklist

\* Required
